### ISSUE TITLE
whois remarks are not available at rdap

### ISSUE THREAD

**Issue by g00g1** [RIPE-NCC/whois#1293]
For example `whois 90.189.112.122` returns this (unnecessary blocks were stripped):

<details><summary>Details</summary>
<p>

```
inetnum:        90.189.112.0 - 90.189.120.255
netname:        WEBSTREAM
descr:          OJSC "Sibirtelecom"
remarks:        Khakassia branch
remarks:        broadband service
country:        RU
remarks:
remarks:        NCC#2007091972
remarks:        INFRA-AW
remarks:
admin-c:        VAS28-RIPE
tech-c:         VAS28-RIPE
tech-c:         VVP8-RIPE
mnt-by:         NSOELSV-NCC
status:         ASSIGNED PA
remarks:
remarks:        Direct reference for the general info on spam
remarks:        In unsoluble cases for the general info on spam,
remarks:        abusing & hacking complaints email abuse@sinor.ru
remarks:
created:        2008-07-21T04:32:24Z
last-modified:  2008-07-21T07:53:15Z
source:         RIPE # Filtered
```

</p>
</details> 

However `curl https://rdap.db.ripe.net/ip/90.189.112.122` returns only whois descr and not remarls:

<details><summary>Details</summary>
<p>

```
{
  "handle" : "90.189.112.0 - 90.189.120.255",
  "startAddress" : "90.189.112.0",
  "endAddress" : "90.189.120.255",
  "ipVersion" : "v4",
  "name" : "WEBSTREAM",
  "type" : "ASSIGNED PA",
  "country" : "RU",
  "parentHandle" : "90.188.0.0 - 90.189.255.255",
  "cidr0_cidrs" : [ {
    "v4prefix" : "90.189.112.0",
    "length" : 21
  }, {
    "v4prefix" : "90.189.120.0",
    "length" : 24
  } ],
  "status" : [ "active" ],
  "entities" : [ {
    "handle" : "NSOELSV-NCC",
    "roles" : [ "registrant" ],
    "objectClassName" : "entity"
  }, {
    "handle" : "VAS28-RIPE",
    "roles" : [ "technical", "administrative" ],
    "objectClassName" : "entity"
  }, {
    "handle" : "VVP8-RIPE",
    "roles" : [ "technical" ],
    "objectClassName" : "entity"
  }, {
    "handle" : "RTNC-RIPE",
    "vcardArray" : [ "vcard", [ [ "version", { }, "text", "4.0" ], [ "fn", { }, "text", "PJSC Rostelecom Technical Team" ], [ "kind", { }, "text", "group" ], [ "adr", {
      "label" : "PJSC Rostelecom\nRussian Federation"
    }, "text", [ "", "", "", "", "", "", "" ] ], [ "email", {
      "type" : "email"
    }, "text", "ripe@rt.ru" ], [ "email", {
      "type" : "abuse"
    }, "text", "abuse@rt.ru" ] ] ],
    "roles" : [ "abuse" ],
    "entities" : [ {
      "handle" : "IE1277-RIPE",
      "roles" : [ "technical", "administrative" ],
      "objectClassName" : "entity"
    }, {
      "handle" : "ROSTELECOM-MNT",
      "roles" : [ "registrant" ],
      "objectClassName" : "entity"
    } ],
    "objectClassName" : "entity"
  } ],
  "remarks" : [ {
    "description" : [ "OJSC \"Sibirtelecom\"" ]
  } ],
  "links" : [ {
    "value" : "https://rdap.db.ripe.net/ip/90.189.112.122",
    "rel" : "self",
    "href" : "https://rdap.db.ripe.net/ip/90.189.112.122"
  }, {
    "value" : "http://www.ripe.net/data-tools/support/documentation/terms",
    "rel" : "copyright",
    "href" : "http://www.ripe.net/data-tools/support/documentation/terms"
  } ],
  "events" : [ {
    "eventAction" : "registration",
    "eventDate" : "2008-07-21T04:32:24Z"
  }, {
    "eventAction" : "last changed",
    "eventDate" : "2008-07-21T07:53:15Z"
  } ],
  "rdapConformance" : [ "cidr0", "rdap_level_0", "nro_rdap_profile_0" ],
  "notices" : [ {
    "title" : "Filtered",
    "description" : [ "This output has been filtered." ]
  }, {
    "title" : "Whois Inaccuracy Reporting",
    "description" : [ "If you see inaccuracies in the results, please visit:" ],
    "links" : [ {
      "value" : "https://rdap.db.ripe.net/ip/90.189.112.122",
      "rel" : "inaccuracy-report",
      "href" : "https://www.ripe.net/contact-form?topic=ripe_dbm&show_form=true",
      "type" : "text/html"
    } ]
  }, {
    "title" : "Source",
    "description" : [ "Objects returned came from source", "RIPE" ]
  }, {
    "title" : "Terms and Conditions",
    "description" : [ "This is the RIPE Database query service. The objects are in RDAP format." ],
    "links" : [ {
      "value" : "https://rdap.db.ripe.net/ip/90.189.112.122",
      "rel" : "terms-of-service",
      "href" : "http://www.ripe.net/db/support/db-terms-conditions.pdf",
      "type" : "application/pdf"
    } ]
  } ],
  "port43" : "whois.ripe.net",
  "objectClassName" : "ip network"
}
```

</p>
</details> 

However, as I can see from PR #1278, rdap should return both remarks and description.

**Comment by eshryane** [comment]
Hello @g00g1 indeed this is fixed in #1278 but it's not live yet. We plan to make a Whois release in the next month or so, and will include this fix.

**Comment by eshryane** [comment]
Hello again @g00g1 this fix is included in Whois 1.108 which we plan to deploy in production on 27th September.

**Comment by eshryane** [comment]
Whois 1.108 is now deployed to production, so this issue should now be fixed.
